stdlib/3.7/simplequeue.pyi

```diff
@@ -0,0 +1,13 @@
+from typing import TypeVar, Generic, Optional
```

Comment:
Seems like this should be in queue.pyi instead. You can use an `if sys.version_info ...` block to make this class accessible only in 3.7.

Reply:
Thanks, I wasn't sure. Fixed.